docs/GettingStartedOnLinux.md

```diff
@@ -0,0 +1,217 @@
+next: android-setup
+---
+
+## Intro
+
+This guide is essentially a beginner-friendly version of the [Getting Started](/react-native/docs/getting-started.html) page for React Native on Linux.
+
+### Platform
+
+For the purposes of this guide, we assume that you're working on Ubuntu Linux 14.04 LTS.
+
+### Prerequisites
+
+Before following this guide, you should have installed the Android SDK and run a successful Java-based "Hello World" app for Android.
+
+Other than that, we should have you covered!
+
+Let's dig in.
+
+### Installing everything
+
+#### Install NodeJS
+
+The first thing you need to do is to install NodeJS, a popular Javascript implementation.
+
+Fire up a terminal and paste these commands in to install NodeJS from APT:
+
+* `sudo apt-get install -y build-essential`
+* `curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -`
+* `sudo apt-get install -y nodejs`
+
+#### Install NPM
+
+After installing NodeJS, you should now have NPM, too, the Node Package Manager.
+
+You can check by typing `which npm` into the terminal.
+
+If have NPM, you'll see something like:
+
+```bash
+$ /usr/local/bin/npm
+```
+
+If you don't have NPM, you'll see:
+
+```bash
+$ which npm
+npm not found
+$
+```
+
+If that's the case, run the following command to install NPM:
+
+`curl http://npmjs.org/install.sh | sh`
+
+#### Install watchman
+
+[watchman](https://facebook.github.io/watchman/docs/install.html) is a tool developed by Facebook for watching the filesystem. For best results, you'll want to install it.
+
+Once again, paste these into your terminal to get up and running:
+
+* `git clone https://github.com/facebook/watchman.git`
+* `cd watchman`
+* `git checkout v4.1.0  # the latest stable release`
+* `./autogen.sh`
+* `./configure`
+* `make`
+* `sudo make install`
+
+#### Install flow
+
+Next up: flow. This is a static type checker for Javascript.
+
+Here's how you install it:
+
+* `wget https://facebook.github.io/flow/downloads/flow-linux64-latest.zip`
+* `unzip flow-linux64-latest.zip`
+* `cd flow`
+* `sudo mv flow /usr/local/bin/flow`
+
+Note: for advanced users, you can put the `flow` executable anywhere on your PATH. It doesn't have to be in `/usr/local/bin`.
+
+#### Install React Native
+
+OK, with all of that out of the way, it's finally time to install React Native.
+
+For most users of this guide, you'll need to use `sudo` to install React Native as root:
+
+* `sudo npm install -g react-native-cli`
+
+For users who installed NPM through other means, you'll likely want to install React Native as a normal user:
+
+* `npm install -g react-native-cli`
+
+## Create Project
+
+Hooray! React Native is installed.
+
+Now it's time to do what we came here to do: make an app!
+
+To create an app project, you'll need to sit down and input this command:
+
+* `react-native init AwesomeProject`
+
+It'll take several minutes, so don't panic if your terminal sits around for a while before spitting out any output.
```

Comment:
I prefer to have one page for everything a beginner will need as it's confusing to jump back and forth in the middle. That is, if we can set this guide up to be entirely "A" then go follow guide "B," or vice versa, great, but at the moment, It would be a an A-B-A sandwich, since the quick start does not get a user up and running on Linux, as noted in [this comment](https://github.com/facebook/react-native/pull/5765/files#r51950623).